package/MDAnalysis/core/AtomGroup.py

```diff
@@ -1926,8 +1953,9 @@ def asphericity(self, **kwargs):
                                               recenteredpos[x, :])
         tensor /= self.totalMass()
         eig_vals = numpy.linalg.eigvalsh(tensor)
-        shape = (3.0 / 2.0) * numpy.sum(numpy.power(eig_vals - numpy.mean(eig_vals), 2)) / numpy.power(
-            numpy.sum(eig_vals), 2)
+        shape = (3.0 / 2.0) * numpy.sum(
```

Comment:
i've seen this commonly for when trying to format longer formulas. But on the other hand this one can be made
shorter and easier to read any way.

``` python
(3./2.) * ((eig_vals - eig_vals.mean())**2).sum() / (eig_vals.sum())**2
```

As a side note using the python power operator `**` for small powers is faster then using `numpy.power`.